### ISSUE TITLE
Inconsistent card pics

### ISSUE THREAD

**Issue by Hello23-Ygopro** [SalvationDevelopment/YGOPro-Salvation-Server#307]
I dont know how much of a priority this is , or how many people actually care about card pics , but another thing i like about Salvation is that it has some missing card pics other English Ygopros dont have , like the [gameboy colored *Egyptian God* cards](https://raw.githubusercontent.com/SalvationDevelopment/YGOPro-Images/master/10000000.jpg) regardless if their not legal or recognized in the official database, they were still seen in alot of official Yugioh games , and i think their the most popular versions most people can relate to cus of their appearance in the anime

It would also be nice to have aliases included in the database for more modified artworks , like the TCG's *Exodia* pieces with the colorful *Spellbinding Circle* background , etc. since theyve been kinda forgotten about, but thats a separate issue

I also cant help notice how inconsistent the card pics are.. i think alot of them are the same from Devpro, but the newer ones from the last few sets i think was made using TCG Editor , but it has its own issues like how it spaces longer card descriptions

Also : 
- Some new card pics have outdated names and descriptions , 
- low quality artwork , 
- will tend to have more mistakes cus they were manually created
- and is inconstant cus they were made by different programs

TL;DR : Have you guys considered using [Magic Set Editor](https://github.com/247321453/MagicSetEditor2) & [DataEditorX](https://github.com/247321453/DataEditorX) instead to make card pics ?
I dont know much about computer programs , but from what i heard its the most productive method to make card pics , theres less chance that there will be mistakes cus it automatically creates the card pics based on their stats in the database , and like TCG Editor it doesnt rely on an internet connection or website like *yugiohcardmaker.net*

**Comment by StormWulf** [comment]
I suggest discussing this issue on our YGOPro-Images repository.
